maui-toc.html

```diff
@@ -523,6 +523,7 @@
 			<li><a href="/maui/DataGrid/unbound-column">Unbound Column</a></li>
 			<li><a href="/maui/DataGrid/export-to-excel">Export to Excel</a></li>
             <li><a href="/maui/DataGrid/export-to-pdf">Export to PDF</a></li>
+            <li><a href="/maui/DataGrid/localization">Export to PDF</a></li>
```

Comment:
The heading title should be **Localization**. Modify the changes.

Reply:
modified